src/js/index.js

```diff
@@ -0,0 +1 @@
+import '../css/styles.css'; //import the initial css file
```

Comment:
also import an asset file to have an example of how we can use images, we can write it for the logo!
```
import image from "../assets/logo.png"

const imageLogo = document.getElementById('logo')
imageLogo.src = image;
```

Reply:
Thanks for your feedback it was added 😀